src/robusta/core/sinks/pagerduty/pagerduty_sink.py

```diff
@@ -48,14 +50,112 @@ def __to_pagerduty_status_type(title: str):
+
+        if finding.subject.node:
+            custom_details["node"] = finding.subject.node
+
+        timestamp = finding.starts_at.astimezone().isoformat()
+
+        for enrichment in finding.enrichments:
+            for block in enrichment.blocks:
+                changes = self.__block_to_changes(block, enrichment)
+                if not changes:
+                    continue
+
+                operation = changes["operation"]
+                if not operation:
+                    continue
+
+                unformatted_texts = self.__to_unformatted_text_for_changes(block)
+                if unformatted_texts:
+                    change_num = 1
+                    for diff_text in unformatted_texts:
+                        custom_details[f"Change {change_num}"] = diff_text
+                        change_num += 1
+
+                description = finding.description
+                changes_count_text = ""
+
+                if operation == K8sOperationType.UPDATE:
+                    if description:
+                        custom_details["Remarks"] = description
+
+                    change_count = changes["change_count"]
+                    changes_count_text = " ({change_count} {changes})".format(change_count=change_count,
+                                                                              changes="change" if change_count == 1 else "changes")
+                elif operation == K8sOperationType.CREATE or operation == K8sOperationType.DELETE:
+                    if description:
+                        custom_details["Remarks"] = f"Resource {operation.value}d"
+
+                summary = f"{finding.service.resource_type} {finding.service.namespace}/{finding.service.name} {operation.value}d in cluster {self.cluster_name}{changes_count_text}"
 
+                body = {
+                    "routing_key": self.api_key,
+                    "payload": {
+                        "summary": summary,
+                        "timestamp": timestamp,
+                        "source": source,
+                        "custom_details": custom_details
+                    },
+                    "links": links
+                }
+
+                headers = {"Content-Type": "application/json"}
+                response = requests.post(self.change_url, json=body, headers=headers)
+                if not response.ok:
+                    logging.error(
+                        f"Error sending message to PagerDuty: {response.status_code}, {response.reason}, {response.text}"
+                    )
+
+    @staticmethod
+    def __send_events_to_pagerduty(self, finding: Finding, platform_enabled: bool):
+        custom_details: dict = {}
+
+        links = []
         if platform_enabled:
-            custom_details["🔎 Investigate"] = finding.get_investigate_uri(self.account_id, self.cluster_name)
+            href = finding.get_investigate_uri(self.account_id, self.cluster_name)
+            links.append({
```

Comment:
I think the local member here isn't needed 
```
            links.append({
                "text": "🔎 Investigate in Robusta",
                "href": finding.get_investigate_uri(self.account_id, self.cluster_name)
            })

```

Reply:
Cleaned the code up